developer-docs-site/docs/tutorials/run-a-local-testnet.md

```diff
@@ -64,34 +98,65 @@ We describe each method below.
       --server-url http://127.0.0.1:8080
 ```
 
-This will start a Faucet running locally without any restrictions to tokens that can be claimed / minted and make the serivce as accessible as the testnet started above. Faucets are stateless services that can be run in parallel.
+This will start a Faucet running locally without any restrictions to tokens that can be claimed and minted. This Faucet service will be as accessible as the testnet youy started above.
```

Comment:
```suggestion
This will start a Faucet running locally without any restrictions to tokens that can be claimed and minted. This Faucet service will be as accessible as the testnet you started above.
```